Example/HelloWorld/HelloWorldLib/Sources/TodoItemRow.swift

```diff
@@ -17,6 +17,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import Down
+import Kingfisher
```

Comment:
Can we add an API usage example too to demonstrate that the indexing also works here?